data/builtin_mount_options.conf

```diff
@@ -8,8 +8,14 @@ vfat_allow=uid=$UID,gid=$GID,flush,utf8,shortname,umask,dmask,fmask,codepage,ioc
 exfat_defaults=uid=$UID,gid=$GID,iocharset=utf8,errors=remount-ro
 exfat_allow=uid=$UID,gid=$GID,dmask,errors,fmask,iocharset,namecase,umask
 
+# the legacy ntfs kernel driver and the ntfs-3g fuse driver
```

Comment:
I think we need a more detailed explanation here. I know it is in the documentation, but nobody is going to read that so having a longer comment in the default config should help.

Reply:
I've added something to the "sample" config file, please have a look if you like it more.